app/main/util/date_filters_validator.py

```diff
@@ -15,14 +15,30 @@ def validate_date_filters(args, browse_consignment=False):
         to_month,
         to_year,
         date_validation_errors,
+        error_field,
     ) = validate_dates(args, browse_consignment=browse_consignment)
 
     if browse_consignment and "date_filter_field" in date_validation_errors:
-        return date_validation_errors, from_date, to_date, date_filters
+        date_filters = {
+            "from_day": args.get("date_from_day"),
```

Comment:
what is this change for?

Reply:
this is to return the actual value entered by user in date filter inputs, earlier they were returned as None and when user receive the date validation error, the inputs were getting blank values instead of values entered by user

Reply:
spoke about shifting this into validate_dates on call

Reply:
updated the code as requested